### PR TITLE
Feature UI

### PR DESCRIPTION
**Pull request by Paola-Xaq**
UI design.

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java [hunk 1]
```diff
@@ -47,29 +71,138 @@
...
+        mainPanel.revalidate();
+    }
+
+    /**
+     * Call the panel for the Button option.
+     *
+     * @param e Action of selected JButton.
+     */
+    @Override
+    public void actionPerformed(ActionEvent e) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/MainInterface.java]: The variable should be final and have a more descriptive name.

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java [hunk 2]
```diff
@@ -47,29 +71,138 @@
...
+        }
+    }
+
+    /**
+     * Adds the style hover and out for JButton.
+     *
+     * @param button to add style.
+     */
+    public BtnStyle mainButtonAction(BtnStyle button) {
+        MouseListener ml = new MouseAdapter() {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/MainInterface.java]: The variable should have a more descriptive name.

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java [hunk 3]
```diff
@@ -47,29 +71,138 @@
...
+            @Override
+            public void mouseEntered(MouseEvent e) {
+                Component c = e.getComponent();
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/MainInterface.java]: The variable should be final and have a more descriptive name.

File: src/main/java/org/fundacion/jala/converter/view/compiler/CodeTextArea.java [hunk 4]
```diff
@@ -109,5 +123,9 @@ public void insertUpdate(final DocumentEvent de) {
     public void removeUpdate(final DocumentEvent de) {
         lineCode.setText(getLinesOfText());
     }
-}
 
+    public void JTreePanelOne(String vals[]) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/compiler/CodeTextArea.java]: The method should have javadoc and its variable should be final.

File: src/main/java/org/fundacion/jala/converter/view/compiler/Console.java [hunk 5]
```diff
@@ -12,28 +12,35 @@
...
         console.setBackground(Color.black);
         console.setForeground(Color.white);
-        console.setEditable(false);
-        console.setPreferredSize(new Dimension(WIDTH_C, HEIGHT_C));
-        console.setBorder(new EmptyBorder(0, 0, 0, 0));
+        console.setEditable(true);
+        console.setBorder(new EmptyBorder(MARGIN, MARGIN, MARGIN, MARGIN));
+        console.setFont(new Font("Courier New", Font.PLAIN, FONT_SIZE));
+        Border border = new LineBorder(Color.WHITE, 0);
+        JScrollPane sp = new JScrollPane(console);
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/compiler/Console.java]: The variable should have a more descriptive name.

File: src/main/java/org/fundacion/jala/converter/view/login/LoginInterface.java [hunk 6]
```diff
@@ -32,93 +44,125 @@
...
+        mainContainer.add(iconContainer);
+        mainContainer.add(userPanel);
+        mainContainer.add(passPanel);
+        mainContainer.setBackground(Color.DARK_GRAY);
+        container = new JPanel();
+        container.setLayout(new FlowLayout(FlowLayout.RIGHT));
+        container.setBackground(Color.DARK_GRAY);
+        CLOSE_BTN = new JButton(new AbstractAction("X") {
+            @Override
+            public void actionPerformed(ActionEvent e) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/login/LoginInterface.java]: The variable should be final and have a more descriptive name.

File: src/main/java/org/fundacion/jala/converter/view/utilities/TxtField.java [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Paola Ximena Aguilar Quiñones
+ */
+package org.fundacion.jala.converter.view.utilities;
+
+import javax.swing.JTextField;
+import javax.swing.border.EmptyBorder;
+import java.awt.*;
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/utilities/TxtField.java]: You should import the specific library and not use *.

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java [hunk 8]
```diff
@@ -47,29 +71,138 @@
...
+        setPosition(buttonsPanel, videoConverterBtn, false);
+        setPosition(buttonsPanel, audioConverterBtn, false);
+        setPosition(buttonsPanel, textExtractorBtn, false);
+        setPosition(buttonsPanel, metaDataBtn, true);
+    }
+
+    /**
+     * Sets positions for FlowLayout.
+     */
+    public void setPosition(JPanel mainPanel, BtnStyle mainBtn, boolean lastBtn) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/MainInterface.java]: This variables should be final.

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java [hunk 9]
```diff
@@ -47,29 +71,138 @@
...
+            positionConstraint.weighty = 1;
+            mainPanel.add(new JLabel(), positionConstraint);
+        }
+        positionYMainBtn += 1;
+    }
+
+    /**
+     * Sets the correct panel according to selected Button.
+     */
+    public void switchPanels(JPanel panel) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/MainInterface.java]: This variable should be final.

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java [hunk 10]
```diff
@@ -47,29 +71,138 @@
...
+            switchPanels(metaDataPanel);
+        }
+    }
+
+    /**
+     * Adds the style hover and out for JButton.
+     *
+     * @param button to add style.
+     */
+    public BtnStyle mainButtonAction(BtnStyle button) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/MainInterface.java]: This variable should be final.

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java [hunk 11]
```diff
@@ -47,29 +71,138 @@
...
+        MouseListener ml = new MouseAdapter() {
+            @Override
+            public void mouseEntered(MouseEvent e) {
+                Component c = e.getComponent();
+                c.setBackground(OVER_BACKGROUND_BTN);
+            }
+
+            @Override
+            public void mouseExited(MouseEvent e) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/MainInterface.java]: The variable should be final and have a more descriptive name.

File: src/main/java/org/fundacion/jala/converter/view/compiler/CompilerMainButtons.java [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+ *  * This class customizes a panel with custom buttons to change language.
+ */
+class CompilerMainButtons extends JPanel {
+    private String newToken;
+    private BtnStyle java;
+    private BtnStyle python;
+    private BtnStyle project;
+    private final int BTN_TYPE_COMPILER = 2;
+
+    protected CompilerMainButtons(String token) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/compiler/CompilerMainButtons.java]: The variable should be final.

File: src/main/java/org/fundacion/jala/converter/view/compiler/CompilerMainButtons.java [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+ */
+package org.fundacion.jala.converter.view.compiler;
+
+import org.fundacion.jala.converter.view.utilities.BtnStyle;
+import javax.swing.JPanel;
+import javax.swing.JButton;
+import java.awt.FlowLayout;
+
+/**
+ *  * This class customizes a panel with custom buttons to change language.
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/compiler/CompilerMainButtons.java]: The second *. should be removed.

File: src/main/java/org/fundacion/jala/converter/view/compiler/CompilerMainButtons.java [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+        java = new BtnStyle("Java", "BtnCompilerJava.png", BTN_TYPE_COMPILER);
+        python = new BtnStyle("Python", "BtnCompilerPython.png", BTN_TYPE_COMPILER);
+        setLayout(new FlowLayout(FlowLayout.LEFT));
+        add(project);
+        add(java);
+        add(python);
+    }
+
+    /**
+     * Button that defines or not java is selected
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/compiler/CompilerMainButtons.java]: The methods should have a description in third person. It seems that the description describes the return and not the method.

File: src/main/java/org/fundacion/jala/converter/view/compiler/CompilerMainButtons.java [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+     *
+     * @return a JButton
+     */
+    public JButton getJava() {
+        return java;
+    }
+
+    /**
+     * Button that defines or not python is selected
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/compiler/CompilerMainButtons.java]: The methods should have a description in third person. It seems that the description describes the return and not the method.

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java [hunk 16]
```diff
@@ -47,29 +71,138 @@
...
+     */
+    public void switchPanels(JPanel panel) {
+        mainPanel.removeAll();
+        mainPanel.add(panel);
+        mainPanel.repaint();
+        mainPanel.revalidate();
+    }
+
+    /**
+     * Call the panel for the Button option.
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/MainInterface.java]: The description should be in third person. Maybe change the Call with Calls.

File: src/main/java/org/fundacion/jala/converter/view/compiler/ProjectTab.java [hunk 17]
```diff
@@ -125,15 +139,21 @@ public CodeTextArea getSelectedPane() {
...
-        tabButton.addActionListener(e1 -> {
+
+        ImageIcon addIcon = new ImageIcon("img/compilerBtn/BtnCloseTab.png");
+        JButton closeBtn = new JButton(addIcon);
+        closeBtn.setOpaque(true);
+        closeBtn.setBackground(Color.WHITE);
+        closeBtn.setFocusPainted(false);
+        Border border = new LineBorder(Color.WHITE, 0);
+        closeBtn.setBorder(border);
+        closeBtn.addActionListener(e1 -> {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/compiler/ProjectTab.java]: The variable e1 should have a more descriptive name.

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java [hunk 18]
```diff
@@ -10,35 +10,59 @@
  */
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/MainInterface.java:10]: You should put the full name of the author

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java [hunk 19]
```diff
@@ -0,0 +1,181 @@
...
+        setBackground(DARK_BACKGROUND_BTN);
+        setFocusPainted(false);
+        setForeground(Color.WHITE);
+        Border border = new LineBorder(Color.WHITE, 0);
+        setBorder(border);
+        setFont(new Font("Barlow", Font.PLAIN, FONT_SIZE_13));
+        setPreferredSize(new Dimension(widthBtn, heightBtn));
+    }
+
+    public void setBtnCompilers(String text, String icon) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java]: The method should have javadocs and the variables should be final.

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java [hunk 20]
```diff
@@ -0,0 +1,181 @@
...
+        Border border = new LineBorder(Color.WHITE, 0);
+        setBorder(border);
+    }
+
+    /**
+     * Sets JButton format for Main Menu Buttons.
+     *
+     * @param text String for the JButton.
+     */
+    public void setBtnMainMenu(String text, String icon) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java]: The variables should be final.

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java [hunk 21]
```diff
@@ -0,0 +1,181 @@
...
+        Border border = new LineBorder(Color.WHITE, 0);
+        setBorder(border);
+    }
+
+    /**
+     * Sets JButton format for Convert Button.
+     *
+     * @param text String for the JButton.
+     */
+    public void setBtnConvert(String text) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java]: The variable should be final.

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java [hunk 22]
```diff
@@ -0,0 +1,181 @@
...
+        setBackground(LIGHT_GRAY_COLOR_BTN);
+        setForeground(Color.DARK_GRAY);
+    }
+
+    /**
+     * Sets JButton format for File Select.
+     *
+     * @param text String for the JButton.
+     */
+    public void setBtnFile(String text) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java]: The variable should be final.

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java [hunk 23]
```diff
@@ -0,0 +1,181 @@
...
+        setBackground(ORANGE_COLOR_BTN);
+        setForeground(Color.WHITE);
+    }
+
+    /**
+     * Sets JButton format for Register.
+     *
+     * @param text String for the JButton.
+     */
+    public void setRegisterBtn(String text) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java]: The variable should be final.

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java [hunk 24]
```diff
@@ -0,0 +1,181 @@
...
+            setBtnCompilers(text, icon);
+        }
+    }
+
+    /**
+     * Sets JButton format for Login.
+     *
+     * @param text String for the JButton.
+     */
+    public void setLoginBtn(String text) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java]: The variable should be final.

File: src/main/java/org/fundacion/jala/converter/view/utilities/ComboStyle.java [hunk 25]
```diff
@@ -0,0 +1,67 @@
...
+        setBorder(new CompoundBorder(BorderFactory.createLineBorder(Color.LIGHT_GRAY, THICK_BORDER),
+                new EmptyBorder(0, MARGIN_COMBOBOX, 0, 0)));
+        setFont(DEFAULT_FONT);
+        setFocusable(false);
+        setBackground(Color.WHITE);
+        setUI(ComboUI.createUI(this));
+        setVisible(true);
+    }
+
+    public ComboStyle(E[] items) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/utilities/ComboStyle.java:56]: the variable should be final.

File: src/main/java/org/fundacion/jala/converter/view/utilities/ComboUI.java [hunk 26]
```diff
@@ -0,0 +1,53 @@
...
+import javax.swing.plaf.basic.BasicComboPopup;
+import javax.swing.plaf.basic.ComboPopup;
+import java.awt.Color;
+
+/**
+ * This class set the UI for the JCombobox.
+ */
+public class ComboUI extends BasicComboBoxUI {
+
+    public static ComboBoxUI createUI(JComponent c) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/utilities/ComboUI.java]: The method should have javadoc, the variable should be final and have a more descriptive name.

File: src/main/java/org/fundacion/jala/converter/view/compiler/CompilerMainButtons.java [hunk 27]
```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Paola Aguilar Quiñones
```
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/view/compiler/CompilerMainButtons.java]: The author's name should have a full name.

File: src/main/java/org/fundacion/jala/converter/view/compiler/CompilerMainButtons.java [hunk 28]
```diff
@@ -0,0 +1,56 @@
...
+        setLayout(new FlowLayout(FlowLayout.LEFT));
+        add(project);
+        add(java);
+        add(python);
+    }
+
+    /**
+     * Button that defines or not java is selected
+     *
+     * @return a JButton
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/compiler/CompilerMainButtons.java:42]: Returns in java docs of this class should be more descriptive

File: src/main/java/org/fundacion/jala/converter/view/compiler/ProjectTab.java [hunk 29]
```diff
@@ -176,8 +196,8 @@ public void changeTitle(final int numberOfClicks) {
      *
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/compiler/ProjectTab.java:196]: Maybe it should be better to say "Gets the plus button to access it" in this java doc

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java [hunk 30]
```diff
@@ -0,0 +1,181 @@
...
+ */
+package org.fundacion.jala.converter.view.utilities;
+
+import javax.swing.JButton;
+import javax.swing.ImageIcon;
+import javax.swing.Icon;
+import javax.swing.SwingConstants;
+import javax.swing.border.Border;
+import javax.swing.border.LineBorder;
+import java.awt.*;
```
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java]: To import libraries you should not use "*"

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java (hunk 24 above)
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java]: The parameter should be final

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java (hunk 23 above)
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java]: The parameter should be final

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java (hunk 22 above)
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java]: The parameter should be final

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java (hunk 21 above)
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java]: The parameter should be final

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java (hunk 20 above)
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java]: The parameters should be final

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java (hunk 19 above)
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java]: The parameters should be final

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java [hunk 31]
```diff
@@ -56,30 +65,36 @@
...
+        container.add(menuConverterType);
+        container.add(settings);
+        setLayout(new BorderLayout());
+        setBorder(new EmptyBorder(0, MARGIN_SPACE, MARGIN_BOTTOM_MAIN_CONTAINER, MARGIN_SPACE));
+        add(videoTitle, BorderLayout.NORTH);
+        add(container, BorderLayout.CENTER);
+        add(btnContainer, BorderLayout.SOUTH);
     }
 
     /**
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java:93]: There is a small error, "with" should be "width", in this java doc

File: src/main/java/org/fundacion/jala/converter/view/utilities/CheckFile.java [hunk 32]
```diff
@@ -27,7 +27,8 @@ public static boolean checkFileSelect(final String file) {
         if (file != null) {
             return true;
         } else {
-            JOptionPane.showMessageDialog(null, "You need select a file", "Error Message", JOptionPane.ERROR_MESSAGE);
+            JOptionPane.showMessageDialog(null, "You need to select a file",
```
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/view/utilities/CheckFile.java]: This line should be 120 characters

File: src/main/java/org/fundacion/jala/converter/view/utilities/ComboUI.java (hunk 26 above)
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/view/utilities/ComboUI.java]: The parameter should be final

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java (hunk 30 above)
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java:19]: You should not import all the libraries, only what is used

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java (hunk 19 above)
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java:165]: Java doc is missing

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java (hunk 31 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java:93]: I'm sorry Dani, but I can't find the error, can you tell me the line?

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java (hunk 18 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/MainInterface.java:10]: Solved!

File: src/main/java/org/fundacion/jala/converter/view/compiler/ProjectTab.java (hunk 29 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/compiler/ProjectTab.java:196]: Solved!

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java (hunk 2 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/MainInterface.java]: Solved!

File: src/main/java/org/fundacion/jala/converter/view/compiler/Console.java (hunk 5 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/compiler/Console.java]: Solved!

File: src/main/java/org/fundacion/jala/converter/view/login/LoginInterface.java (hunk 6 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/login/LoginInterface.java]: Solved!

File: src/main/java/org/fundacion/jala/converter/view/compiler/CodeTextArea.java (hunk 4 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/compiler/CodeTextArea.java]: Solved!

File: src/main/java/org/fundacion/jala/converter/view/compiler/CodeTextArea.java [hunk 33]
```diff
@@ -109,5 +123,9 @@ public void insertUpdate(final DocumentEvent de) {
     public void removeUpdate(final DocumentEvent de) {
         lineCode.setText(getLinesOfText());
     }
-}
 
+    public void JTreePanelOne(final String vals[]) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/compiler/CodeTextArea.java:132]: Method should have javadoc.

File: src/main/java/org/fundacion/jala/converter/view/login/LoginInterface.java [hunk 34]
```diff
@@ -32,93 +44,125 @@
...
+        mainContainer.add(iconContainer);
+        mainContainer.add(userPanel);
+        mainContainer.add(passPanel);
+        mainContainer.setBackground(Color.DARK_GRAY);
+        container = new JPanel();
+        container.setLayout(new FlowLayout(FlowLayout.RIGHT));
+        container.setBackground(Color.DARK_GRAY);
+        CLOSE_BTN = new JButton(new AbstractAction("X") {
+            @Override
+            public void actionPerformed(ActionEvent mouseEvent) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/login/LoginInterface.java:154]: Does this method should not have javadoc?

File: src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java [hunk 35]
```diff
@@ -0,0 +1,184 @@
...
+        setBackground(DARK_BACKGROUND_BTN);
+        setFocusPainted(false);
+        setForeground(Color.WHITE);
+        Border border = new LineBorder(Color.WHITE, 0);
+        setBorder(border);
+        setFont(new Font("Barlow", Font.PLAIN, FONT_SIZE_13));
+        setPreferredSize(new Dimension(widthBtn, heightBtn));
+    }
+
+    public void setBtnCompilers(final String text, final String icon) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/utilities/BtnStyle.java:174]: This method should have javadoc.

**Review by edsonarios — APPROVED** [review]
Good job

File: src/main/java/org/fundacion/jala/converter/view/login/LoginInterface.java (hunk 34 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/login/LoginInterface.java:154]: **

> Does this method should not have javadoc?

I don't think and ActionPerformed inside a method should have a JavaDoc

File: src/main/java/org/fundacion/jala/converter/view/compiler/CodeTextArea.java (hunk 33 above)
Comment (Paola-Xaq) [reply src/main/java/org/fundacion/jala/converter/view/compiler/CodeTextArea.java:132]: Solved!

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java (hunk 31 above)
Comment (danielaSantaCruz) [reply src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java:93]: Line 101 pao

**Review by Raymundogs — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java (hunk 31 above)
Comment (danielaSantaCruz) [reply src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java:93]: Sorry it was my mistake, it's ok :)

**Review by danielaSantaCruz — APPROVED** [review]
Great job Pao! :)

**Pull request merged**